Simplenote/src/main/java/com/automattic/simplenote/NotesActivity.java

```diff
@@ -1413,9 +1408,10 @@ public void onConfigurationChanged(@NonNull Configuration newConfig) {
                 }
 
                 invalidateOptionsMenu();
-            // Go to NoteEditorActivity if note editing was fullscreen and orientation was switched to portrait
-            } else if (mNoteListFragment.isHidden() && mCurrentNote != null) {
+                // Go to NoteEditorActivity if a note was selected and orientation was switched to portrait
```

Comment:
```suggestion
            // Go to NoteEditorActivity if a note was selected and orientation was switched to portrait
```

It looks like autoformat added an extra indent to this comment.